File: android/app/src/main/java/com/smartexpense/android/data/remote/dto/request/ExpenseRequestDto.java
package com.smartexpense.android.data.remote.dto.request;

public class ExpenseRequestDto {
    private double amount;
    private String category;
    private String photoUrl;
    private String caption;
    private String expenseDate;

    public ExpenseRequestDto(double amount, String category, String photoUrl, String caption, String expenseDate) {
        this.amount = amount;
        this.category = category;
        this.photoUrl = photoUrl;
        this.caption = caption;
        this.expenseDate = expenseDate;
    }
}

File: android/app/src/main/kotlin/com/smartexpense/android/data/remote/dto/request/ExpenseRequestDto.kt
package com.smartexpense.android.data.remote.dto.request

import com.google.gson.annotations.SerializedName

data class ExpenseRequestDto(
    @SerializedName("amount") val amount: Double,
    @SerializedName("category") val category: String,
    @SerializedName("photoUrl") val photoUrl: String,
    @SerializedName("caption") val caption: String,
    @SerializedName("expenseDate") val expenseDate: String
)
